config.json

```
@@ -12,13 +12,23 @@
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 2
+    "indent_size": 2,
+    "ace_editor_language": "delphi",
```

```suggestion
    "ace_editor_language": "pascal",
```
Delphi does not appear to be supported by ace.